### PR TITLE
Add files and instructions for using Mountpoint with Docker

### PR DESCRIPTION
**Pull request by mheshwg**
---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and I agree to the terms of the [Developer Certificate of Origin (DCO)](https://developercertificate.org/).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +80,7 @@ For the alpha release, additional configuration is required to use Mountpoint fo
 
 ## Contributing
 
-We welcome contributions to Mountpoint for Amazon S3! Please see [CONTRIBUTING.md](doc/CONTRIBUTING.md) for more information on how to report bugs or submit pull requests, [LOGGING.md](doc/LOGGING.md) for details on how to capture logging data for bug reports, and [BENCHMARKING.md](doc/BENCHMARKING.md) for performance regression testing.
```
Comment (monthonk) [inline README.md:62]: I don't think you intend to change this line.

**Review by monthonk — CHANGES_REQUESTED** [review]
Only one minor comment and it should be good.

File: README.md (hunk 1 above)
Comment (mheshwg) [reply README.md]: Done!. thank you.

File: README.md [hunk 2]
```diff
@@ -47,6 +47,27 @@ Then run the client, specifying the directory in which your S3 bucket should be
 
 The client will run in the background by default, and the `~/mnt` directory now gives access to the objects in your S3 bucket.
 
+### Mountpoint for Amazon S3 using Docker
+
+First, build the docker image using the below command.
+
+    cd mountpoint-s3/docker
```
Comment (monthonk) [inline README.md:54]: It's `mountpoint-s3/docker` here but the actual Dockerfile is in `docker`. So, either we move that file to the right path or change the command here.

**Review by monthonk — COMMENTED** [review]
Sorry, one more comment, I missed that one last time.

**Review by monthonk — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (monthonk) [reply README.md:54]: We have offline conversation that we're assuming user are running the command from outside project root directory. It's the same assumption on [how to compile section](https://github.com/awslabs/mountpoint-s3/blob/2f07b5ec976aad219e2d1445909eca3cf0b4976a/README.md?plain=1#L36-L37), so I think this is fine. This is a bit confusing that we also use the same name for sub project.

Comment (monthonk) [reply README.md:54]: But feel free to suggest the change if you still think we should update it,

Comment (mheshwg) [reply README.md:54]: Thank you Monthonk; I am aligned. Keeping the current version as is.

**Review by dannycjones — APPROVED** [review]
LGTM, tested instructions locally also.

**Comment by dannycjones** [comment]
Merged, thanks for the contribution @mheshwg! 🚀

**Pull request merged**